network-api/networkapi/wagtailpages/factory/bannered_campaign_page.py

```diff
@@ -35,11 +35,11 @@ def generate(seed):
     reseed(seed)
 
     try:
-        bannered_campaign_namespace = WagtailPage.objects.get(title='campaigns')
-        print('Campaigns namespace exists')
+        bannered_campaign_index_page = WagtailPage.objects.get(title='campaigns')
```

Comment:
we probably want to call this `campaign_index_page`, since it can contain various child types.